### DISCUSSION TITLE
se(効果音)募集

### DISCUSSION THREAD

**Discussion by i7-FF7FC** [beta1139/ff7fc-develop#86] [General]
今作ってるイベでまだ作ってないseがあるので、
良かったらお暇な人作ってみませんか？

ニブル山の泉で天然マテリアが輝いているシーンのバックで
うっすらフォーンと鳴ってる神秘的な音です:musical_note:
締切厳守！期限無し:triumph:

**Comment by ponyexs** [comment]
では早速。

[bgs_3oggs.zip](https://github.com/beta1139/ff7fc-develop/files/5976011/bgs_3oggs.zip)

-----
　●**BGS_魔コウの泉.ogg**

音量は同シーンのBGM「1-14-Underneath_the_Rotting_Pizza.ogg」を基準にやや小さめにしています。
<br><br>



それとついでなのでこれも。

　**●魔晄炉駆動音BGS(TEST.ogg
　●魔晄エネルギー冷却凝縮システムBGS(TEST.ogg**

ニブル山の魔晄炉内の環境音です。

が…
魔晄の泉の音ほど近づけることはできませんでした。🙏

魔晄炉内のBGM「1-03-Makou_Reactor.ogg」と一緒に鳴らしてみたら
音の質が近いのでどうにも分かりづらかったです… (^^;

…ので音量を大きめにしてみたのですが…

これもし実際に使ったら…うるさいだけかも。😷

**Reply by i7-FF7FC** [reply]
なんという速さ！すばらしす＆感謝:joy:
まだ実際にイベには嵌めてませんが、
魔晄炉の音は、自作のポッド覗くシーンの音(原作は特にありません)が
イマイチだったのでこれ良いかも。明日試してみます！

**Reply by i7-FF7FC** [reply]
泉の音嵌めてみました。
音の大きさ含めて完璧でございました:clap:
もう今後se全部投げようかと思ったぐらいw

魔晄炉の音は、散策中ずっと流すと確かにちょっと大きいような気もしますね。
まだそのイベには行ってないので、ちょっと分からないところもありますが、
実際に流すシーン(回想後半に魔晄炉を訪れる場面)では、
bgmは無しなので埋もれることはないかと。

ところで、いきなり問答無用でexeが立ち上がらなくなってしばらく格闘してたのですが、
原因はmapdata\setting.txtをいじった時に
nothingじゃなくてnotingとタイポってたのが原因でした。
いやぁ焦った・・備忘録代わりにここにメモ。

**Reply by ponyexs** [reply]
魔晄の泉はうまく当てはまったようで良かった良かった🤣です
ありがとうございます

…が、魔晄炉のほうの音はどちらも単品でしばらく聴いていると
だんだんイライラしてくる😖ので
不快な（耳障りな）音の類なのかもしれません。
原作のような静かだけど存在感のある音には
なかなかできないですねー　困った😷
<br>

なお頼まれてもいない音（曲）を勝手に作るのは
ゲーム制作時に素材の選択肢が増えればいいなとの考えからですので、
不要であれば放っておいてもらえば結構です
（これまでも実際に作っても7割程度を没にしていますし）

**Reply by i7-FF7FC** [reply]
魔晄炉のマップ自体は、そんなに長居する場所でもないので、
しばらく聞き続ける配慮はそこまで要らないかも知れませんけどねw

寧ろ今回のポッド覗く時とか、瞬間的にピンポイントで使った場合は、
逆に不快な音の方が原作的なサイコ臭も感じられて良かったなと。

マップ中bgmとして鳴らす場合は、単純に音量を少しだけ
下げれば良い気もしますが…はて:thinking:
